README2.md

```diff
@@ -0,0 +1 @@
+# I exit on a different branch to the master branch 
```

Comment:
Why did you use this variable? maybe consider x y and z

Reply:
great job!